pkgs/development/python-modules/cron-descriptor/default.nix

```
@@ -1,32 +1,34 @@
 {
   lib,
-  python,
-  buildPythonPackage,
   fetchFromGitHub,
+  buildPythonPackage,
+  unittestCheckHook,
   mock,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "cron_descriptor";
```

```suggestion
  pname = "cron-descriptor";
```

The `pname` should be normalized like that.